internal/process/queue.go

```diff
@@ -96,8 +131,43 @@ func (q *Queue) worker(queue workqueue.RateLimitingInterface, process func(key s
 				}
 
 				queue.Forget(key)
+				log.Infof("item for %s has been processed, no retry, element forgotten", id)
+
 				return false
 			}()
 		}
 	}
 }
+
+func (q *Queue) logAndUpdateWorkerTimes(key string, name string, log logrus.FieldLogger) {
+	// log time
+	now := time.Now()
+	lastTime, ok := q.workerExecutionTimes[name]
+	if ok {
+		log.Infof("execution - worker %s last execution time %s, executed after %s", name, lastTime, now.Sub(lastTime))
```

Comment:
Is this specifier appropriate for time value?

Reply:
Yes, the logs are formatted as follows: "executed after 161.541µs"